src/wymeditor/core.js

```diff
@@ -81,9 +81,7 @@ jQuery.extend(WYMeditor, {
     STRINGS             - An array of loaded WYMeditor language pairs/values.
     SKINS               - An array of loaded WYMeditor skins.
     NAME                - The "name" attribute.
-    INDEX               - A string replaced by the instance index.
     WYM_INDEX           - A string used to get/set the instance index.
-    BASE_PATH           - A string replaced by WYMeditor's base path.
```

Comment:
These two seem to not be required now that popup dialogs don't load any scripts.